### PR TITLE
[ACCEL] Update ora2 repository add settings

### PR DESCRIPTION
**Pull request by arsentur**
**Description:**
Change ora2 edx repository to rg repository
Add settings for ora

**Youtrack:**
https://youtrack.raccoongang.com/issue/ACCEL-75

### CODE REVIEW

File: requirements/edx/base.txt [hunk 1]
```diff
@@ -39,6 +38,7 @@ git+https://github.com/edx-solutions/xblock-drag-and-drop-v2@v2.1.6#egg=xblock-d
 -e git+https://github.com/edx-solutions/xblock-google-drive.git@138e6fa0bf3a2013e904a085b9fed77dab7f3f21#egg=xblock-google-drive
 git+https://github.com/open-craft/xblock-poll@add89e14558c30f3c8dc7431e5cd6536fff6d941#egg=xblock-poll==1.5.1
 git+https://github.com/edx/xblock-utils.git@v1.1.1#egg=xblock-utils==1.1.1
+-e git+https://github.com/raccoongang/edx-ora2@rg-v.2.1.17-accel#egg=rg_ora2
```
Comment (idegtiarov) [inline requirements/edx/base.txt]: Please add a tag to the egg as well.

File: requirements/edx/development.txt [hunk 2]
```diff
@@ -42,6 +41,7 @@ git+https://github.com/edx-solutions/xblock-drag-and-drop-v2@v2.1.6#egg=xblock-d
 -e git+https://github.com/edx-solutions/xblock-google-drive.git@138e6fa0bf3a2013e904a085b9fed77dab7f3f21#egg=xblock-google-drive
 git+https://github.com/open-craft/xblock-poll@add89e14558c30f3c8dc7431e5cd6536fff6d941#egg=xblock-poll==1.5.1
 git+https://github.com/edx/xblock-utils.git@v1.1.1#egg=xblock-utils==1.1.1
+-e git+https://github.com/raccoongang/edx-ora2@rg-v.2.1.17-accel#egg=rg_ora2
```
Comment (idegtiarov) [inline requirements/edx/development.txt]: ditto

File: requirements/edx/testing.txt [hunk 3]
```diff
@@ -40,6 +39,7 @@ git+https://github.com/edx-solutions/xblock-drag-and-drop-v2@v2.1.6#egg=xblock-d
 -e git+https://github.com/edx-solutions/xblock-google-drive.git@138e6fa0bf3a2013e904a085b9fed77dab7f3f21#egg=xblock-google-drive
 git+https://github.com/open-craft/xblock-poll@add89e14558c30f3c8dc7431e5cd6536fff6d941#egg=xblock-poll==1.5.1
 git+https://github.com/edx/xblock-utils.git@v1.1.1#egg=xblock-utils==1.1.1
+-e git+https://github.com/raccoongang/edx-ora2@rg-v.2.1.17-accel#egg=rg_ora2
```
Comment (idegtiarov) [inline requirements/edx/testing.txt]: ditto

File: requirements/edx/base.txt (hunk 1 above)
Comment (arsentur) [reply requirements/edx/base.txt]: Done

File: requirements/edx/development.txt (hunk 2 above)
Comment (arsentur) [reply requirements/edx/development.txt]: Done

File: requirements/edx/testing.txt (hunk 3 above)
Comment (arsentur) [reply requirements/edx/testing.txt]: Done

**Pull request merged**